cmd/ctr/utils.go

```diff
@@ -79,3 +85,68 @@ func waitContainer(events execution.ContainerService_EventsClient, id string, pi
 		}
 	}
 }
+
+func forwardAllSignals(containers execution.ContainerServiceClient, id string) chan os.Signal {
+	sigc := make(chan os.Signal, 128)
+	catchAll(sigc)
+	go func() {
+	Signals:
+		for s := range sigc {
+			var sig syscall.Signal
+			for sigStr, sigN := range signalMap {
```

Comment:
since you are getting the actual signal in `s` you don't need to do this type of check and parse, just assign `Signal: s.(uint32)`